Naggum.Assembler/Program.fs

```diff
@@ -0,0 +1,52 @@
+﻿module Naggum.Assembler.Program
+
+open System
+open System.IO
+open System.Reflection
+open System.Reflection.Emit
+
+type ReturnCode =
+    | Success = 0
+    | Error = 1
+    | InvalidArguments = 2
+
+let printUsage () =
```

Comment:
Make `printUsage` and `printError` methods private; they have no use outside of this assembly. The same for `ReturnType`.